1.9/api/versioning.md

```diff
@@ -0,0 +1,31 @@
+---
+post_title: Versioning
+menu_order: 2
+---
+
+The DC/OS API is backed by many loosely coupled components, some of which are standalone projects and some of which were designed exclusively for DC/OS. Because of this, several types of versioning mechanisms are supported: component, route, and resource versioning.
+
+To learn how to formulate a specific API call, see the component API reference documentation for that route.
+
+## Component Versioning
```

Comment:
If you make these H1s (aka `#`) they will show up on the left hand side "In this post".